Test/Tests/Network/SingleHostWithUtils.Tests.ps1

```diff
@@ -158,11 +157,11 @@ Test-WithRetries 3 {
 
             Write-Log 'Getting VM NetAdapter Information'
             $VMNetInfo = Get-RemoteNetAdapterInformation -Session $Session `
-                -AdapterName $Testenv.System.AdapterName
+                -AdapterName $Session.DataAdapterName
```

Comment:
Could you use $testbed variable instead of $session? It's kind of confusing.

Reply:
Right now we have big mess with Sessions/Testbeds. I am removing using sessions in tests on PR's dedicated to each Test. Will it be ok?

Reply:
Done